CHANGELOG.md

```diff
@@ -1,3 +1,26 @@
+0.10.0 (Apr 1x, 2021)
+====================
```

Comment:
let's put in the date today here. I am going to do the version bump in `_version.py` by directly pushing to master.

Reply:
done!